frontend/src/components/home/heroSection/Hero.tsx

```diff
@@ -1,62 +1,45 @@
-import TopBar from "../../_shared/TopBar";
-import { Organizations, FindData, Groups } from "./Options";
-import SearchForm from "./SearchForm";
-import Stats, { StatsProps } from "./Stats";
+import { ChevronRightIcon } from "@heroicons/react/20/solid";
+import Link from "next/link";
+import SearchBar from "@components/search/SearchBar";
+import Video from "./Video";
+import InfoCards from "./InfoCards";
+import PartnersCarousel from "@components/_shared/PartnersCarousel";
 
-export default function Hero({ stats }: { stats: StatsProps }) {
+export default function Hero() {
   return (
-    <section className="sm:grid sm:grid-rows-frontpage-hero">
-      <section className="row-start-1 row-span-3 col-span-full">
-        <div
-          className="bg-cover bg-center bg-no-repeat bg-black flex flex-col h-full"
-          style={{
-            backgroundImage: "url('/images/backgrounds/HomeHero.png')",
-            minHeight: "95vh",
-          }}
-        >
-          <TopBar />
-          <div className="grid lg:grid-cols-2 mx-auto items-center grow custom-container">
-            <div className="col-span-1">
-              <h3 className="text-xs font-medium pb-2 uppercase text-gray-400 tracking-widest sm:text-base lg:text-sm xl:text-base">
-                Quality Data ready to Integrate
-              </h3>
-              <h1 className="text-6xl font-black text-white">Find and Share</h1>
-              <h1 className="text-6xl font-black text-cyan-500">
-                Quality Data
-              </h1>
-              <h3 className="text-xl text-gray-300 py-6">
-                At Portal, we have over thousands of datasets for free and a
-                Premium Data Service for additional or customised data with
-                guaranteed updates.
-              </h3>
-              <div className="sm:max-w-lg sm:text-center lg:text-left lg:mx-0">
-                <SearchForm />
-              </div>
-              <div className="sm:max-w-lg py-4 lg:pb-12 sm:py-0">
-                <Stats
-                  datasetCount={stats.datasetCount}
-                  orgCount={stats.orgCount}
-                  groupCount={stats.groupCount}
-                />
-              </div>
-            </div>
+    <section className="pb-[96px] pt-[64px]">
+      <div className="container">
+        <div className="flex items-center gap-x-16 xl:items-start">
+          <div className="w-full">
+            <Link
+              href="#"
+              className="flex w-fit items-center rounded-[14px] bg-[#F3F4F6] p-2"
```

Comment:
I think changing this p-2 to p-1.5 makes it look more like the mockup
![image](https://github.com/user-attachments/assets/b1fa2c0c-7f03-4069-bbfb-acb9d3b7119a)

Reply:
fixed